NAMESPACE

```diff
@@ -121,22 +117,33 @@ import(circlize)
 import(config)
 import(cowplot)
 import(data.table)
-import(dbplyr)
 import(digest)
 import(dplyr)
+import(forcats)
 import(g3viz)
 import(ggbeeswarm)
 import(ggplot2)
 import(ggpubr)
 import(ggrepel)
 import(ggsci)
 import(ggthemes)
+import(glue)
 import(grid)
+import(gridExtra)
```

Comment:
What is using gridExtra? Maybe we should rework that function to avoid another dependency? It is also probably already a sub-dependency of one of the packages like Complexeatmap or circlize

Reply:
Currently, there's only one package that imports gridExtra and it is `comp_report`. It's the wrapper function for all the sample-level fancy plots. It uses `gridExtra` to arrange all plots and output a pdf. I can look into if the same output can be achieved without the dependency on this package.

Reply:
Ok nice! Didn't know that `ggpubr` could achieve the same thing. I'll leave it as is for now.